openpype/lib/events.py

```diff
@@ -246,14 +157,38 @@ def __init__(self, topic, func, order):
         topic_regex = re.compile(topic_regex_str)
         self._topic_regex = topic_regex
 
-        self._wrapper_func = _WrappedFunc(func)
-        self._enabled = True
+        # Callback function prep
+        if isinstance(func, weakref_partial):
+            partial_func = func
+            (name, path) = _get_func_info(func.get_func())
+            func_ref = None
+            expect_args = False
+            expect_kwargs = False
```

Comment:
This seems odd - a regular `partial` can still expect `args` and `kwargs` just fine?